ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -153,7 +153,7 @@ function integration::bazel_with_emulators() {
   # - GOOGLE_CLOUD_CPP_SPANNER_DEFAULT_ENDPOINT
   io::log_h2 "Running Spanner integration tests"
   bazel "${verb}" "${args[@]}" --test_tag_filters=integration-test \
-    google/cloud/spanner/...
+    --test_timeout=3600 google/cloud/spanner/...
```

Comment:
Why do we need that?  Do the BUILD-file values not apply?

Reply:
It's set to 600 above, so I need to override it here.

Also, `eternal` implies a timeout of 3600s, so this is setting it to the equivalent of eternal.

https://docs.bazel.build/versions/master/test-encyclopedia.html#role-of-the-test-runner

Reply:
... of course, I could remove the `--test_timeoout` above and always rely on the per-test size setting... that may be better.